tests/document/instance.py

```diff
@@ -3149,6 +3172,23 @@ class Person(Document):
 
         person.update(set__height=2.0)
 
+    @needs_mongodb_v26
+    def test_push_with_position(self):
+        """Ensure that push with position works properly for an instance."""
+        class BlogPost(Document):
+            slug = StringField()
+            tags = ListField(StringField())
+
+        blog = BlogPost()
+        blog.slug = "ABC"
+        blog.tags = ["python"]
+        blog.save()
+
+        blog.update(push__tags__0=["mongodb", "code"])
```

Comment:
Is it possible to push a single item with the positional operator? For example `push__tags__0='mongodb'`? If so, that behavior should be tested, too.